scripts/metric_reporter/main.py

```diff
@@ -29,47 +32,92 @@
 logger = logging.getLogger(__name__)
 
 
-def main(config_file: str = "config.ini") -> None:
+def main(
+    config_file: str = "config.ini",
+    output_csv: bool | None = None,
+    update_bigquery: bool | None = None,
+) -> None:
     """Run the Metric Reporter.
 
     Args:
         config_file (str): Path to the configuration file.
-                           Defaults to 'ecosystem-test-scripts/config.ini'.
+        output_csv (bool | None): Whether to output CSV files. Overrides config file if provided.
+        update_bigquery (bool | None): Whether to update BigQuery tables. Overrides config file if provided.
```

Comment:
Not sure if we need to add a "Default: False" to both of these (since if I'm reading this correctly the default behavior would be to not output CSV files OR update BigQuery).

(although I see below we have a handy warning message if we hit this case! 😍 )

> "The metric reporter will not perform any action. Use the --output_csv or --update_bigquery flags."

Reply:
I have the `config.ini.sample` setup for both these values to be True at the minute, when I set default with the command line it overrides the config, hence 'None'